sa/sa_test.go

```diff
@@ -64,7 +64,7 @@ func initSA(t *testing.T) (*SQLStorageAuthority, clock.FakeClock, func()) {
 	fc := clock.NewFake()
 	fc.Set(time.Date(2015, 3, 4, 5, 0, 0, 0, time.UTC))
 
-	sa, err := NewSQLStorageAuthority(dbMap, dbMap, fc, log, metrics.NoopRegisterer, 1)
+	sa, err := NewSQLStorageAuthority(dbMap, dbMap, nil, nil, fc, log, metrics.NoopRegisterer, 1)
```

Comment:
Probably best to initialize both of these by default, probably with a `mockRedisClient` which just keeps state in-memory and can be inspected and manipulated directly by the tests. Then existing tests can be minimally updated to assert things about redis state.